compiled_starters/haskell/.codecrafters/run.sh

```
@@ -0,0 +1,11 @@
+#!/bin/sh
+#
+# This script is used to run your program on CodeCrafters
+#
+# This runs after .codecrafters/compile.sh
+#
+# Learn more: https://codecrafters.io/program-interface
+
+set -e # Exit on failure
+
+exec $(stack path --local-install-root)/bin/codecrafters-bittorrent-exe "$@"
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)